### PR TITLE
changed bootcamp to kicad software

### PR DESCRIPTION
**Pull request by scholvat**

### CODE REVIEW

File: docs/bootcamp/electrical.md [hunk 1]
```diff
@@ -2,70 +2,93 @@
...
-- Learn where to find documentation and how to understand it.
+In this bootcamp you will learn how to do the following in KiCad:
+
+- search digikey and select a component
+- manage bill of materials (BOM) and the WARG component library
+- make a symbol or find symbol libraries
+- edit a schematic
+- find a footprint and map it to the part
+- place and route a pcb
+- export gerbers
```
Comment (edenfrosst) [inline docs/bootcamp/electrical.md]: Capitalize sentences, acronyms (PCB, etc), punctuate end of lines.

**Review by edenfrosst — APPROVED** [review]
All looks good. Just a few changes for consistency and formality. Might be a bit tough for a beginner, but it's roughly in line with the difficulty of the mech bootcamp. Might need to link some KiCAD tutorials, if people are struggling too much with it.

File: docs/bootcamp/electrical.md [hunk 2]
```diff
@@ -2,70 +2,93 @@
...
+## 1. ZeroPilot DC/DC Buck Regulator
 
 ### Background
 
-The tracking antenna helps to ensure our ground station keeps in contact with the plane at all times by pointing antennas at the plane, using data from the ground station. To do this, the tracking antenna has 2 servos that point the antennas, an arduino to do computations and basic control, and multiple sensors to help in calibration. 
-The power circuit allows the antenna to be powered by either a battery or from a power cable, and simplifies the wiring to the servos and arduino.
+In 2017 WARG started work on the new autopilot pcb, called ZeroPilot. With the new board comes a lot of tasks involving designing new circuits, converting circuits from Diptrace (our old software) to KiCad, and fixing bugs in various circuits. ZeroPilot is based off of STM microprocessors. It has a main processor called the autopilot controller which is a Cortex-M7 processor which handles most of our calculations. The board also has a secondary processor as a failsafe, which is called the safety controller. It allows us to confidently test new autopilot code since we can always switch into safety mode if there is an autopilot software error.
```
Comment (edenfrosst) [inline docs/bootcamp/electrical.md]: PCB

File: docs/bootcamp/electrical.md [hunk 3]
```diff
@@ -2,70 +2,93 @@
...
+
+### Resources
+
+- [Zeropilot repo](https://github.com/UWARG/ZeroPilot-HW)
+- [KiCad library repo](https://github.com/UWARG/ZeroPilot-HW)
+- [Digikey to find components](https://www.digikey.com/)
+- [KiCad Software](http://kicad-pcb.org/)
+- [KiCad Documentation](http://kicad-pcb.org/help/documentation/)
+- [KiCad getting started guide](http://docs.kicad-pcb.org/4.0.6/en/getting_started_in_kicad.pdf)
+-  bootcamp channel on  [slack](https://uwarg.slack.com) to ask questions to team leads
```
Comment (edenfrosst) [inline docs/bootcamp/electrical.md]: Capitalize

File: docs/bootcamp/electrical.md [hunk 4]
```diff
@@ -2,70 +2,93 @@
...
+11. Export the gerbers into a zip file. These files are what are sent to the manufacturer to print the boards and stencils.
+
+## Submitting the bootcamp
+
+Create a pull request (PR) to WARG github. Make sure to title your bootcamp PR in the format, "Bootcamp: {name here}". There should be 2 PRs, one for each repo (ZeroPilot and WARG-KiCad). The ZeroPilot PR should also include a zip file containing the manufacturing files. A team lead will review the PR within a few days and reply with approval or with some comments. The team lead will close the PR and not merge it when it is complete.
+
+
+### Resources
+
+- [Zeropilot repo](https://github.com/UWARG/ZeroPilot-HW)
```
Comment (edenfrosst) [inline docs/bootcamp/electrical.md]: CamelCase ZeroPilot

**Comment by scholvat** [comment]
@ianjfrosst I realize this might be a bit tough for a beginner to electronics so if you have any other bootcamp ideas I would be willing to hear them.

**Pull request merged**